Detail/ObjectViewController.swift

```diff
@@ -53,9 +53,16 @@ class ObjectViewController: UIViewController {
 }
 
 extension ObjectViewController: ObjectViewDataSource {
-    func componentsInObjectView(_ objectView: ObjectView) -> [Component] {
+    func components(in objectView: ObjectView) -> [Component] {
         return components
     }
+
+    func customComponentView(for component: Component,in objectView: ObjectView) -> UIView {
```

Comment:
Maybe this should return UIView? and return nil if it doesn't know how to handle the case?

Reply:
Sure. Should it display an empty `UIView` if it does not now how to handle the case?

Reply:
I think it should ignore it in that case